router-tests/feature_flags_test.go

```diff
@@ -20,7 +20,7 @@ func TestFeatureFlags(t *testing.T) {
 				Query: `{ employees { id productCount } }`,
 			})
 			require.Empty(t, res.Response.Header.Get("X-Feature-Flag"))
-			require.JSONEq(t, `{"errors":[{"message":"field: productCount not defined on type: Employee","path":["query","employees","productCount"]}],"data":null}`, res.Body)
```

Comment:
So if there is a propagated error, the data shouldn't be null—the property should not exist at all?

Reply:
All errors before query execution e.g parsing, validation should not contain data property. https://spec.graphql.org/draft/#sec-Data